dbos/_context.py

```diff
@@ -358,7 +358,9 @@ class EnterDBOSWorkflow(AbstractContextManager[DBOSContext, Literal[False]]):
     def __init__(self, attributes: TracedAttributes) -> None:
         self.created_ctx = False
         self.attributes = attributes
-        self.is_temp_workflow = attributes["name"] == "temp_wf"
+        self.is_temp_workflow = (attributes["name"] == "temp_wf_sync") or (
```

Comment:
Hmm can we not change the name from `temp_wf` to `temp_wf_sync`? It's kind of a breaking change because our monitoring system and the cloud console have to filter out `temp_wf` for traces. We can add `temp_wf_async` but I'd keep the sync one simply `temp_wf`

Reply:
Certainly easy to change *one* of the temp wf functions to be just `temp_wf`. Will see if I can do both